### PR TITLE
Admin/readme

### PR DESCRIPTION
**Pull request by elleshadow**
# Description drawing

Adding draft of the README.md file.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?

- [ ] console.log()
- [ ] dev tools
- [ ] functionality check through web browser

# Checklist:

- [ ] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..)
- [ ] We have performed a self-review of our code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have deleted all comments/console.log() after functionality and understanding is final
- [ ] I have added my other members are reviewers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+3. Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
+4. Refine your collaboration skills
+
+### Technologies Used
+* Git 
+* HTML
+* CSS
+* JavaScript
+
+## Applicaiton Features
```
Comment (JoshMallery) [inline README.md:27]: Please correct the Spelling of Application.

**Review by JoshMallery — CHANGES_REQUESTED** [review]

**Review by JoshMallery — CHANGES_REQUESTED** [review]
There is one Type on line 27

**Review by JoshMallery — APPROVED** [review]
Reviewed and Approved on my end.

**Review by elleshadow — COMMENTED** [review]
Good catch.

**Review by mrlandino — COMMENTED** [review]
Hey Y'all,

This all looks fantastic. Nice work. We will definitely add more when we get closer to the end of the project.

**Pull request merged**